server/src/main/java/com/google/musicanalysis/site/YoutubeServlet.java

```diff
@@ -96,9 +96,13 @@ protected void doGet(HttpServletRequest req, HttpServletResponse res)
         JsonObject likedVideoRes;
         JsonArray videos;
         YoutubeGenres genreAnalysis = new YoutubeGenres();
+        // needed to keep track of likedMusicHistory for heat map
```

Comment:
this comment doesn't really make sense to me, since it's not "keeping track" of `likedMusicHistory` but being used by it. `likedMusicHistory` also isn't part of this file, so that could come of as confusing to someone only looking at this file. consider rephrasing to something like "used to determine the position of a newly retrieved video in comparison to already retrieved videos while obtaining more videos in segments"